src/main/java/com/example/SOPSbackend/controller/BasicController.java

```diff
@@ -0,0 +1,59 @@
+package com.example.SOPSbackend.controller;
+
+import com.example.SOPSbackend.exception.InternalValidationException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.validation.FieldError;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BasicController {
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<Object> handleValidationException(MethodArgumentNotValidException ex) {
+        Map<String, String> errors = new HashMap<>();
+        for(ObjectError error : ex.getBindingResult().getAllErrors()) {
+            String field = ((FieldError)error).getField();
+            String message =  error.getDefaultMessage();
+            errors.put(field, message);
+        }
+
+        return ResponseEntity.unprocessableEntity()
+                .body(new HashMap<String, Object>() {{
+                    put("success", false);
```

Comment:
Jak niżej.

Reply:
Niestety nie. Map.of ma sygnaturę typów: K k1, V v1, K k2, V v2....